cli/ballerina-cli/src/main/resources/cli-help/ballerina-openapi.help

```diff
@@ -33,24 +33,24 @@ OPTIONS
             the output will be written onto the same directory from where the command is run.
 
         (--mode) <mode-type>
-            Mode type can be service or client. Generate a ballerina service and client only
+            Mode type can be service or client. Generate a Ballerina service and client only
             according to the mode. This is optional input.
 
         (-m| --module) <module-name>
```

Comment:
Shall we remove this `--module ` flag in the help text(Line 39-40), because it is not going to support the openAPI tool?

Reply:
Removed in c0da303